src/data/faq.json

```diff
@@ -2239,19 +2472,29 @@
             ],
             "V": [
                 {
-                    "term": "Validity of an EU digital COVID certificate",
+                    "term": "Validity of an EU Digital COVID Certificate",
                     "anchor": "validity",
                     "description": "Basically, we distinguish between <ul><li>the validity of the digital certificate (the technical validity) and</li><li>the validity of the vaccination protection (the effective validity).</li></ul>The digital certificate has an issue date and a technical expiration date (so does your passport and PA). The validity of the certificate ends 1 year after issuance. <br/><br/> <b>Vaccination Certificate</b> <br/><br/>In the certificate data is transported, in this case information about vaccination. This vaccination took place on a date X and is valid from then (according to the current state of scientific knowledge) e.g. in Germany 1 year. After that the vaccination protection has expired. We call the duration of the vaccination protection 'effective (vaccination) validity'. So the effective validity of the vaccination counts from the last vaccination date (date X).  <br/><br/> A certificate is valid until either the technical validity or the effective validity expires."

```

Comment:
For English (US) punctuation rules:

add comma after "e.g."
"e.g. in Germany 1 year." => "e.g., in Germany 1 year."

add comma after "So"
"So the effective validity"  => "So, the effective validity"

Reply:
Will be solved in the next commit, thanks!